builders/xcm/xcm-transactor.md

```diff
@@ -0,0 +1,194 @@
+
+## Building an XCM with the XCM-Transactor Pallet {: #build-xcm-xcmtransactor-pallet}
+
+This guide covers building an XCM message for remote executions using the XCM-Transactor pallet, specifically with the `transactThroughDerivative` function. The steps to use the `transactThroughDerivativeMultilocation` function are identical, but instead of the currency ID, you specify a multilocation for the fee token.
+
+!!! note
+    You need to ensure that the call you are going to execute remotely is allowed in the destination chain!
+
+### Checking Prerequisites {: #xcmtransactor-check-prerequisites}
+
+To be able to send the extrinsics in Polkadot.js Apps, you need to have:
+
+ - An [account](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/accounts){target=_blank} with [funds](https://docs.moonbeam.network/builders/get-started/networks/moonbase/#get-tokens){target=_blank}
+ - The account from which you are going to send the XCM through the XCM-Transactor pallet must also be registered in a given index to be able to operate through a derivative account of the sovereign account. The registration is done through the root account (SUDO in Moonbase Alpha), so contact us to get it registered
+ - Remote calls through XCM-Transactor require `xcUNIT` tokens to pay for the execution fees, which is the [XC-20](/builders/xcm/xc20/){target=_blank} representation of the Alphanet relay chain token `UNIT`. You can acquire some by swapping for DEV tokens (Moonbase Alpha's native token) on [Moonbeam-Swap](https://moonbeam-swap.netlify.app){target=_blank}, a demo Uniswap-V2 clone on Moonbase Alpha
+
+![Moonbeam Swap xcUNITs](/images/builders/xcm/xc20/xtokens/xtokens-1.png)
+
+To check your `xcUNIT` balance, you can add the XC-20 to MetaMask with the following address:
+
+```
+0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080
+```
+
+You can check the [XC-20](/builders/xcm/xc20/#calculate-xc20-address){target=_blank} page to learn how to calculate this address.
+
+### XCM-Transactor Transact Through Derivative Function {: #xcmtransactor-transact-through-derivative}
+
+In this example, you'll build an XCM message to execute a remote call in the relay chain from Moonbase Alpha through the `transactThroughDerivative` function of the XCM-Transactor pallet.
+
+If you've [checked the prerequisites](#xcmtransactor-check-prerequisites), head to the extrinsic page of [Polkadot JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/extrinsics){target=_blank} and set the following options:
+
+1. Select the account from which you want to send the XCM. Make sure the account complies with all the [prerequisites](#xcmtransactor-check-prerequisites)
+2. Choose the **xcmTransactor** pallet
+3. Choose the **transactThroughDerivative** extrinsic
+4. Set the destination to **Relay**, to target the relay chain
+5. Enter the index of the derivative account you've been registered to. For this example, the index value is `42`
```

Comment:
This derived account at index 42 is not going to be the same one that someone running the tutorial would be using correct? So their index is probably going to be different? I think this could be made clearer.